src/Maestro/Types/V1/Transactions.hs

```diff
@@ -73,3 +77,65 @@ instance IsTimestamped PaginatedUtxo where
 
 instance HasCursor PaginatedUtxo where
   getNextCursor = _paginatedUtxoNextCursor
+
+-- | Complete transaction Details when queried by its hash.
+data TxDetails = TxDetails
+  { _txDetailsTxHash :: !TxHash
+  -- ^ Transaction hash (identifier)
+  , _txDetailsBlockHash :: !BlockHash
+  -- ^ Hash of the block which includes the transaction.
+  , _txDetailsBlockAbsoluteSlot :: !Word64
+  -- ^ Absolute slot of the block which includes the transaction
+  , _txDetailsBlockHeight :: !BlockHeight
+  -- ^ Block height (number) of the block which includes the transaction
+  , _txDetailsBlockTimestamp :: !Word64
```

Comment:
Prolly some other type here as mentioned earlier.

Reply:
Here we can use `NominalDiffTime`.